### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by suvarna-ratna**
waiting for feedback

### CODE REVIEW

File: javascript/javascript1/week2/flightbooking.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        if(gender == 'male'){
+            return `Lord ${firstname} ${surname}`
+        }
+           
+    } else if (gender == 'female'){
+        return `Lordess ${firstname} ${surname}`
+    }
+        else
+        return `${firstname} ${surname}`
+
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/flightbooking.js:20]: Good job. Just check if firstname surname are empty

File: javascript/javascript1/week2/EventApplication.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const todayDate = new Date()
+const day = todayDate.getDay()
+const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+function getEventWeekday(eventInDays) {
+    const eventDay = (eventInDays + day)%7
+    return eventDay
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/EventApplication.js]: This is an index of the day. You can return the day instead `return weekday[eventDay]`

File: javascript/javascript1/week2/student-manager.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+const class07Students = [];
+function addStudentToClass(studentName) {  
+if(studentName === "") {
```
Comment (zuhairtaha) [inline javascript/javascript1/week2/student-manager.js]: What if `studentName = "    "` logn empty string. 
- hint: use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)

**Review by zuhairtaha — CHANGES_REQUESTED** [review]
Good job 👍
I see that sometimes you use semicolons and sometimes no. 
Try to follow a rule. To always use it or not. And I recommend using it.

File: javascript/javascript1/week2/flightbooking.js (hunk 1 above)
Comment (suvarna-ratna) [reply javascript/javascript1/week2/flightbooking.js:20]: okay

File: javascript/javascript1/week2/EventApplication.js (hunk 2 above)
Comment (suvarna-ratna) [reply javascript/javascript1/week2/EventApplication.js]: okay.. i fixed it.. thanks

File: javascript/javascript1/week2/student-manager.js (hunk 3 above)
Comment (suvarna-ratna) [reply javascript/javascript1/week2/student-manager.js]: okay.. thanks .. fixed

**Pull request merged**